port/win32/omrsysinfo.c

```diff
@@ -62,6 +62,15 @@ omrsysinfo_set_number_entitled_CPUs(struct OMRPortLibrary *portLibrary, uintptr_
 	return;
 }
 
+void
+omrsysinfo_set_number_active_CPUs(struct OMRPortLibrary *portLibrary, uintptr_t number)
+{
+	Trc_PRT_sysinfo_set_number_active_CPUs_Entered();
+	portLibrary->portGlobals->activeCPUs = number;
+	Trc_PRT_sysinfo_set_number_active_CPUs_Exit(number);
+	return;
```

Comment:
you do not need this return

Reply:
Removed the return in `omrsysinfo_set_number_entitled_CPUs` as well as the returns of both in `/unix/omrsysinfo.c` for consistency as well.